storage.bs

```diff
@@ -178,7 +178,18 @@ larger <a>site storage quota</a>. Factors such as navigation frequency, recency
 bookmarking, and <a href="#persistence">permission</a> for {{"persistent-storage"}} can be used as
 indications of "popularity".
 
+The <dfn export>application cache site storage usage</dfn> for an origin is a rough estimate of the amount
```

Comment:
Nit in these definitions: _origin_ is used but not defined. Either say "an origin _origin_" or later say "the origin's site storage unit"